eslint-config-lib/README.md

```diff
@@ -1,13 +1,39 @@
 # eslint-config-inrupt-lib
 Eslint and prettier configs
 
-
 ## Installation
 
+If Node <= 14:
+
+1. `npm install --save-dev @inrupt/eslint-config-base`
+1. `npm install --save-dev @inrupt/eslint-config-lib`
+1. `npx install-peerdeps @inrupt/eslint-config-lib`
+1. `npx install-peerdeps @inrupt/eslint-config-base`
+1. Add `extends: ['@inrupt/eslint-config-lib']` to your .eslintrc.js file.
+
+If Node >= 16:
```

Comment:
Should 16.x be the primary development version? Currently we've nvmrc of 12 and 14 in different packages.

Reply:
I'd be in favor of defaulting to 16, as long as we continue to run tests for all LTS versions.

That's documented [here](https://github.com/inrupt/public-documentation/blob/master/coding-conventions/javascript-coding-standards.md#tools-and-quality-practices), and is a little out of date as it recommends 12:

Reply:
Right, that's not currently the case across our repos, so we should probably create a ticket to enforce this